scripts/start

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+require('dotenv').config()
+
+const version = require('../package.json').version
+const execute = require('child_process').execSync
+
+const clientPort = process.env.ARAGON_PORT || process.env.REACT_APP_PORT || 3000
+
+if (
+  process.env.ARAGON_ETH_NETWORK_TYPE === 'main' &&
+  !process.env.ARAGON_DEFAULT_ETH_NODE
+) {
+  console.log(
+    '⚠️  You are connecting the Aragon network dashboard to mainnet but have not specified an Ethereum node.'
+  )
```

Comment:
Is this necessary right now?

Reply:
I don’t think `ARAGON_DEFAULT_ETH_NODE` will be needed since we are going to fetch the data from Connect / The Graph.